frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -204,7 +203,7 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
       const patch = [
         {
           op: 'add',
-          path: `/metadata/labels/${nodeLabel}`,
+          path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
```

Comment:
Try taking openshift-storage form constants file as with multi-cluster coming in the picture the hardcoding will not work.

Reply:
Here its not needed. Refer https://github.com/openshift/console/pull/2359#discussion_r319112478

Reply:
Understood but still needs to be handled when multicluster comes in picture.